test/single-workspace-tests/extension/extensions/OrganizeImportsOnSaveExtension.test.ts

```diff
@@ -53,7 +53,8 @@ describe('OrganizeImportsOnSaveExtension', () => {
         });
     });
 
-    it('should remove an unused import on save', async () => {
+    it('should remove an unused import on save', async function () {
+        this.timeout(4000);
```

Comment:
These tests consistently take more than 2s on my 2013 MBA to run, resulting in inaccurate test failures. I took the liberty of allowing them up to 4s to run, which solved the issue for me. These then *fail* locally (grmbl) but work in CI, so…